zng/docs/spec.md

```diff
@@ -386,6 +412,7 @@ is interpreted as follows:
 | `bstring`  | variable |  UTF-8 byte sequence with `\x` escapes           |
 | `enum `    | variable |  UTF-8 bytes of enum string                      |
 | `ip`       | 4 or 16  |  4 or 16 bytes of IP address                     |
+| `port`     | 2      |  2 byte unsigned int          |
```

Comment:
Nits:
1. `s/2 byte unsigned int/unsigned int of length 2/` for consistency with the other integer types.
2. Add spaces to keep the table pretty.

Reply:
1. Strictly speaking, the `...of length...` notation seems to be used for the ones that list `variable` in the `N` column, whereas most of the ones that have a static number in the `N` column lead off with a reference to that static number like @alfred-landrum's suggestion was doing, albeit with a slightly different wording, i.e. if I was trying to follow that pattern I might have landed on `2 bytes of unsigned int`. But that reads a little odd to me, so I've just gone with your suggestion.

2. You're certainly right about the spaces, so I've fixed that to.